src/code/padmgr.c

```diff
@@ -6,9 +6,21 @@
 #include <ultra64/controller.h>
 #include <padmgr.h>
 
+/*
+Absent: padmgr_force_stop_ON     padmgr.o 
+Absent: padmgr_force_stop_OFF     padmgr.o 
+padmgr_PakConnectCheck     padmgr.o 
+padmgr_HandleDoneReadPadMsg     padmgr.o 
+padmgr_ConnectCheck     padmgr.o 
+Absent: padmgr_RequestPadData_NonLock     padmgr.o 
+Absent: padmgr_ClearPadData     padmgr.o 
+Absent: padmgr_Create     padmgr.o 
+Absent: padmgr_isConnectedController     padmgr.o 
+*/
+
 s32 D_8012D280 = 1;
 
-OSMesgQueue* PadMgr_LockGetControllerQueue(PadMgr* padmgr) {
+OSMesgQueue* padmgr_LockSerialMesgQ(PadMgr* padmgr) {
```

Comment:
imo, Queue is more descriptive than Q, even if this was the name used in animal crossing we usually make changes to them if they aren't sufficent. Also we want to keep the name style consistent. So I would do Padmgr_LockSerialMesgQueue (or PadMgr like it already was since its technically two words)